Resources/Prototypes/Entities/Objects/Devices/translators.yml

```diff
@@ -0,0 +1,217 @@
+- type: entity
+  id: TranslatorUnpowered
+  parent: [ BaseItem ]
+  name: Translator
+  description: "Translates speech."
+  components:
+  - type: Sprite
+    sprite: Objects/Devices/translator.rsi
+    state: icon
+    layers:
+      - state: icon
+      - state: translator
+        shader: unshaded
+        visible: false
+        map: [ "enum.ToggleVisuals.Layer", "enum.PowerDeviceVisualLayers.Powered" ]
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ToggleVisuals.Toggled:
+        enum.ToggleVisuals.Layer:
+          True: { visible: true }
+          False: { visible: false }
+  - type: HandheldTranslator
+    enabled: false
+
+- type: entity
+  id: Translator
+  parent: [ TranslatorUnpowered, PowerCellSlotMediumItem ]
+  suffix: Powered
+  components:
+  - type: PowerCellDraw
+    drawRate: 1
+
+- type: entity
+  id: TranslatorEmtpy
+  parent: [ Translator ]
+  suffix: Empty
+  components:
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  id: CanilunztTranslator
+  parent: [ TranslatorEmtpy ]
+  name: Canilunzt translator
+  description: "Translates speech between Canilunzt and Galactic Common."
+  components:
```

Comment:
very end-of-PR but it'd be really nice if this was localised in the locale files. Allows downstreams to translate it as needed and if they want to change the name of the Vulp language they can do so in the locale files.

Reply:
Entity Prototypes are inherently localized following a specific pattern, and don't need to be written as such in their file.